src/site/content/en/blog/vitals-business-impact/index.md

```diff
@@ -129,16 +129,37 @@ to monitor progress and impact across teams.
 - Experimented with different rendering techniques
 (for example, SSR LCP element vs SSR above the fold content vs Full client-side rendering).
 
+### Redbus
+
+CWV fixes contributed to 80-100% mobile conversion rates(mCVR) and significant domain ranking uplift across Redbus’s global market properties.
+
+{% Img 
+src="image/akX10MMTHtVhXYWSgqNknjak5KQ2/1988aZMNJ7viKhj8U3aS.png", 
+alt="192% domain ranking uplift in Columbia", 
+width="800", height="360" %}
+
+#### Techniques
+
+- Fixing slots for page components and removal of unoptimized tag insertion scripts helped [optimize CLS](/optimize-cls).
+- [3P Script Optimization](/controlling-third-party-scripts) and building microservices with single responsibility principle reduced TTI & TBT significantly.
+
+#### Key learnings
+
+- Reducing CLS from 1.65 to 0 significantly uplifted their domain rankings globally.
+- Reducing TTI + TBT by ~100% boosted the mCVR by 80-100% across global properties.
+- [Using RUM tools](/vitals-measurement-getting-started) helped capture the real world performance metrics in lower tier markets.
+- Adopting a [performance culture](/performance-budgets-101) is very important to avoid regression. This also enables tech gains such as optimized code, faster releases, componentization and fewer production issues.
```

Comment:
```suggestion
- Adopting a [performance culture](/performance-budgets-101) is very important to avoid regression. This also improves team productivity thanks to optimized code, faster releases, and fewer production issues.
```